### ISSUE TITLE
Unrecognized keyword arguments: `[:adaptivealg]`

### ISSUE THREAD

**Issue by frankschae** [SciML/DiffEqNoiseProcess.jl#110]
We should update the tests in https://github.com/SciML/DiffEqNoiseProcess.jl/blob/master/test/sde_adaptivedistribution_tests.jl
because they currently throw a warning because `adaptivealg` is not a recognized `kwarg` of `solve`

https://github.com/SciML/StochasticDiffEq.jl/blob/master/src/solve.jl#L320

https://github.com/SciML/DiffEqNoiseProcess.jl/blob/eb57dded61255d48815c7c64d89fa5034f831cde/test/RSwM1_test.jl#L4 seems to be the best way to do it.

**Comment by ChrisRackauckas** [comment]
Yes that's how it's supposed to be done. That's some really old syntax.
